data.json

```
@@ -1289,6 +1292,14 @@
             ],
             "description": "The free and open-source wiki software package that powers Wikipedia."
         },
+	    {
+  "name": "PyMC",
+  "desc": "A Python library for Bayesian statistical modeling and probabilistic machine learning. Beginner-friendly with 'good first issue' labels.",
+  "url": "https://github.com/pymc-devs/pymc",
+  "tags": ["python", "bayesian", "statistics", "machine-learning"],
+  "beginner-friendly-label": "good first issue"
+}
+
         {
             "name": "CiviWiki",
```

```suggestion
            "description": "The free and open-source wiki software package that powers Wikipedia."
        },
	{
            "name": "PyMC",
            "link": "https://github.com/pymc-devs/pymc",
            "label": "beginner friendly",
            "technologies": [
                "Python"
            ],
            "description": "A Python library for Bayesian statistical modeling and probabilistic machine learning. Beginner-friendly with 'good first issue' labels."
        },
        {
            "name": "CiviWiki",
```
Fix a few categories and the label given, along with some formatting.

---

{
  "name": "PyMC",
  "link": "https://github.com/pymc-devs/pymc",
  "label": "beginner friendly",
  "technologies": ["Python"],
  "description": "A Python library for Bayesian statistical modeling and probabilistic machine learning. Beginner-friendly with 'good first issue' labels."
}